data/seedChats.js

```diff
@@ -2,7 +2,7 @@ const faker = require('faker');
 const { models } = require('../models/index');
 const randomMinMax = require('../utils/helperFunctions.js').randomMinMax;
 const remainder = require('../utils/helperFunctions.js').remainder;
-const TICKET_STATUSES = require('../models/TICKET_STATUSES.js');
+const TICKET_STATUSES = require('../utils/TICKET_STATUSES.js');
```

Comment:
What is the need/convention for capitalisation of these consts? I know you just moved them in this case but I'm interested.

Reply:
CC: @ConConRob